chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -353,6 +353,12 @@ developer.missingAttestationChangeRequest.report.subject=Semi-annual Attestation
 developer.missingAttestationChangeRequest.report.salutation=Hello %s,
 developer.missingAttestationChangeRequest.report.paragraph=Emails were sent to the following developers:
 
+#URL Uptime Report
+urlUptime.report.filename=URL Uptime
+urlUptime.report.subject=CHPL URL Uptime Report
+urlUptime.report.heading=CHPL URL Uptime Report
+urlUptime.report.paragraph1.body=See attached file 
```

Comment:
Should we have more content here? Maybe not needed for this ticket, but could be worth floating the idea to get it on the PO's radar